scikits/odes/sundials/cvode.pyx

```diff
@@ -1,5 +1,8 @@
 from cpython.exc cimport PyErr_CheckSignals
+from collections import namedtuple
+from enum import IntEnum
```

Comment:
Ok, we need to add for python2.7 support: install https://pypi.python.org/pypi/enum34/
I have to check if this works